content/blog/2025/05/2025-04-25-bridges-update.md

```diff
@@ -0,0 +1,23 @@
++++
+date = "2025-05-14"
+title = "An Update on Bridges"
+
+[taxonomies]
+author = ["Thib"]
+category = ["Bridges"]
++++
+
+Following up on our [Crossroads blog post](/blog/2025/02/crossroads/) we’re giving an update on bridges. The Foundation indeed can’t afford to keep maintaining the bridges anymore and needs to focus its very little resources on its core missions. 
+
+Element manifested their interest in helping us keep the following public instances of the bridges online.
+
+| Instance | Platform | Project |
```

Comment:
I wish we could point to some docs about how to use the specific instance? We don't exactly have m.org hs docs on the site yet. Is there anything else suitable?